src/functions-reference/unbounded_continuous_distributions.Rmd

```diff
@@ -110,14 +115,38 @@ with parameters $\mu = 0$ and $\sigma = 1$.
 The standard normal (location zero, scale one) log probability density
 of y.
 
-### Sampling Statement
+<!-- real; std_normal_cdf; (reals y); -->
+\index{{\tt \bfseries std\_normal\_cdf }!{\tt (reals y): real}|hyperpage}
 
-`y ~ ` **`std_normal`**`(reals y)`
+`real` **`std_normal_cdf`**`(reals y)`<br>\newline
+The cumulative standard normal distribution of y; std_normal_cdf will
+underflow to 0 for $y$ below -37.5 and overflow to 1 for $y$ above 8.25;
+the function `Phi_approx` is more robust in the tails.
+
+<!-- real; std_normal_lcdf; (reals y); -->
+\index{{\tt \bfseries std\_normal\_lcdf }!{\tt (reals y): real}|hyperpage}
+
+`real` **`std_normal_lcdf`**`(reals y)`<br>\newline
+The log of the cumulative standard normal distribution of y; std_normal_lcdf
+will underflow to $-\infty$ for $y$ below -37.5 and overflow to 0 for $y$
+above 8.25; see above for discussion of `Phi_approx` as an alternative.
+
+<!-- real; std_normal_lccdf; (reals y); -->
+\index{{\tt \bfseries std\_normal\_lccdf }!{\tt (reals y): real}|hyperpage}
+
+`real` **`std_normal_lccdf`**`(reals y)`<br>\newline
+The log of the complementary cumulative standard normal distribution of y;
+std_normal_lccdf will overflow to 0 for $y$ below -37.5 and underflow to
+$-\infty$ for $y$ above 8.25; see above for discussion of `Phi_approx` as
```

Comment:
We should be suggesting `log1m(Phi_approx(...))` as an approximation to the lccdf because the naive `log(1 - Phi_approx(...))` will round too agressively.

A link rather than "above" would be helpful as I couldn't find the discussion of approximations.

Reply:
OK, I've reworked the text incorporating your suggestion, and avoided the generic pointing to above.